src/Nethermind/Nethermind.Mev/Data/MevBundle.cs

```diff
@@ -18,21 +18,31 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading;
 using Nethermind.Core;
 using Nethermind.Core.Crypto;
 using Nethermind.Int256;
+using Nethermind.Mev.Source;
+using Nethermind.Serialization.Rlp;
 
 namespace Nethermind.Mev.Data
 {
-    public class MevBundle : IEquatable<MevBundle>
+    public class MevBundle : IEquatable<MevBundle>, IComparable<MevBundle>
```

Comment:
keep record classes as close to POCO / records, maybe even try to use the record feature from latest C# (never used it so do not know). do not add business logic to data classes